eng/WpfArcadeSdk/tools/ExtendedAssemblyInfo.targets

```diff
@@ -40,6 +40,26 @@
       <AssemblyMetadata Include="Commit" Condition="'$(SourceRevisionId)'!=''">
         <Value>$(SourceRevisionId)</Value>
       </AssemblyMetadata>
+
+      <!-- 
+        Crossgen wants to detect when an assembly is built from C++/CLI sources, 
+        since it is unable to function correctly today (ostensibly due to bugs). Since
+        /clr:pure is deprecated and WPF will eventually switch over to /clr (i.e., IJW),
+        Crossgen will eventually (i.e., in the near-term) fail to work on C++/CLI 
```

Comment:
Note: Crossgen does not support IJW today. There are no plans to support them so far